packages/fxa-auth-client/lib/client.ts

```diff
@@ -2532,6 +2547,17 @@ export default class AuthClient {
     return this.sessionDelete('/recovery_phone', sessionToken, {}, headers);
   }
 
+  /**
+   * Disables 2FA Protection on the account.
```

Comment:
This description and return type is not accurate, this endpoint removes the recovery phone, not 2FA